.github/workflows/instrumentation-matrix-nightly.yaml

```
@@ -0,0 +1,313 @@
+  notify-google-chat:
+    runs-on: ubuntu-latest
+    needs: [publish-matrix-summary, open-issue-on-failure]
+    # Quiet on success. Run on any failure signal: per-cell failures the
+    # report counted, OR a catastrophic emission/heavy crash before reports
+    # were written. `!cancelled()` lets this fire even when
+    # open-issue-on-failure itself errored (intermittent gh-cli 5xx) —
+    # ISSUE_URL just stays empty — while still dying cleanly on cancellation
+    # rather than lingering queued like a bare `always()` would.
+    if: |
+      !cancelled() && (
+        needs.publish-matrix-summary.outputs.has_failures == 'true' ||
```

actionlint (1.7.12)

[error] property "full-emission-matrix" is not defined in object type {open-issue-on-failure: {outputs: {issue_url: string}; result: string}; publish-matrix-summary: {outputs: {counts_json: string; has_failures: string}; result: string}}

(expression)